en/sim_sih/index.md

```diff
@@ -105,7 +105,7 @@ The vehicle can be observed moving in jMAVSim, and on the QGC _Fly_ view.
 
 SIH can be run as SITL (Software-In-The-Loop) from v1.14.
 What this means is that the simulation code is executed on the laptop/computer, similar to Gazebo or jMAVSim.
-In this case you don't need the flight controller hardware.
+In this case you don’t need the flight controller hardware.
```

Comment:
I would have reverted this. We don't use the curly apostrophes anywhere else in the docs.